releases/release-6.0.0-dmr.md

```diff
@@ -0,0 +1,576 @@
+    - Fix a bug that TiKV panics when performing profiling on ARM platform [#10658](https://github.com/tikv/tikv/issues/10658)
+    - Fix a bug that TiKV might panic when running 2 years or more [#11940](https://github.com/tikv/tikv/issues/11940)
+    - Fix the compilation issue on the ARM64 architecture caused by missing SSE instruction set [#12034](https://github.com/tikv/tikv/issues/12034)
+    - (dup: release-5.3.1.md > Bug fixes> TiKV)- Fix the issue that deleting an uninitialized replica might cause an old replica to be recreated [#10533](https://github.com/tikv/tikv/issues/10533)
+    - Fix the bug that stale messages causes TiKV to panic [#12023](https://github.com/tikv/tikv/issues/12023)
+    - Fix the issue that undefined behavior (UB) might occur in TsSet conversions [#12070](https://github.com/tikv/tikv/issues/12070)
+    - Fix a bug that replica reads might violate the linearizability [#12109](https://github.com/tikv/tikv/issues/12109)
+    - Fix the potential panic issue that occurs when TiKV performs profiling on Ubuntu 18.04 [#9765](https://github.com/tikv/tikv/issues/9765)
+    - Fix the issue that tikv-ctl returns an incorrect result due to its wrong string match [#12049](https://github.com/tikv/tikv/pull/12049)
+    - Fix the issues of intermittent packet loss and out of memory (OOM) caused by the overflow of memory metrics [#12160](https://github.com/tikv/tikv/issues/12160)
+    - Fix the potential issue of reporting TiKV panics when exiting TiKV [#12231](https://github.com/tikv/tikv/issues/12231)
+
++ PD
+
+    - Fix the issue that the operator creates steps with meaningless Joint Consensus [#4534](https://github.com/tikv/pd/pull/4534)
+    - Fix a bug that the TSO revoking might get stuck when closing the PD client [#4550](https://github.com/tikv/pd/pull/4550)
+    - Fix the issue that the Region scatterer scheduling might cause lost peers [#4570](https://github.com/tikv/pd/pull/4570)
+    - Fix the issue that `Duration` fields of `dr-autosync` cannot be dynamically configured [#4653](https://github.com/tikv/pd/pull/4653)
+
++ TiFlash
+
+    - Fix the issue of TiFlash panic when the memory limit is enabled [#3902](https://github.com/pingcap/tiflash/issues/3902)
+    - Fix the issue that expired data is recycled slowly [#4146]([https://github.com/pingcap/tiflash/issues/](https://github.com/pingcap/tiflash/issues/3902)4146)
+    - Fix the potential issue of TiFlash panic when `Snapshot` is applied simultaneously with multiple DDL operations [#4072]([https://github.com/pingcap/tiflash/issues/](https://github.com/pingcap/tiflash/issues/3902)4072)
+    - Fix the potential query error after adding columns under heavy read workload [https://github.com/pingcap/tiflash/issues/3967](https://github.com/pingcap/tiflash/issues/3967)
+    - Fix the issue that the `SQRT` function with a negative argument returns `NaN` instead of `Null` [#3598] [https://github.com/pingcap/tiflash/issues/3598](https://github.com/pingcap/tiflash/issues/3598)
```

Comment:
```suggestion
    - Fix the potential query error after adding columns under heavy read workload [#3967](https://github.com/pingcap/tiflash/issues/3967)
    - Fix the issue that the `SQRT` function with a negative argument returns `NaN` instead of `Null` [#3598](https://github.com/pingcap/tiflash/issues/3598)
```